### ISSUE TITLE
Hey guy, I'm bothering about play RTSP on Browser/HTML5

### ISSUE THREAD

**Issue by ryanemax** [lsowen/cam-record#1]
# These many way I've tried.
## 1.Directly Play by VLC

It's a good try, and work fine with VLC.
I using: 
- npapi-vlc to play rtsp://xxxxxx in firefox/IE.
- libvlc to play rtsp://xxxx in Android/IOS
- WebChimera.js for Electron/NW.js

But the Problem is that, chrome has already stop the support for npapi since semptember, 2015.
And the Firefox declare that they will also stop the support for npapi in the beginning of 2016.

Opera/Edge/Firefox/Chrome will support ppapi for a long term.
But until now, There isn't any ppapi-vlc support.
## 2.Convert RTSP to RTMP via Server, and then play with ppapi(flash)

![like this](http://www.umediaserver.net/images/overview.png)
I'm very interested in Golang and RESTful API.
I stared go-rest-json & gin & raml & gorm for a long time.
That's why I found here: https://github.com/lsowen/cam-record/

I didn't found the readme.md. could you intro your project for short?

**Comment by lsowen** [comment]
@ryanemax Good point, I will update the project with a short README which explains what it does and how to run it.
